### ISSUE TITLE
Solutions for challenges

### ISSUE THREAD

**Issue by hdashnow** [swcarpentry/r-novice-gapminder#23]
Many students asked for solutions to the challenges. This is particularly important when we don't get time to discuss the later challenges with the whole class. It's also handy for new instructors teaching the first workshop.

I suggest putting a link to a page of challenges and solutions at the end of each lesson.

**Comment by aammd** [comment]
thanks @liz-is  and @hdashnow ! I like the idea of including answers. 
@liz-is in your PR you mentioned 

> Note that moving the 08-plot-ggplot2 solutions to the end and formatting them in the same way as the other solutions means that some plots are no longer produced and included in the html - is this a problem?

Could you elaborate a bit? were these plots originally inside some challenge chunks?

**Comment by sritchie73** [comment]
This is fixed by Pull Request #41

**Comment by liz-is** [comment]
@aammd Yes, so there were originally solutions in R chunks just after the challenges. When I moved these to the end I formatted them as the other existing solutions were formatted (I don't know if this style has a name?):
```
> ~~~ {.r}
> #code goes here
> ~~~
```
But then the code wasn't run when the file was knitted so the plots didn't appear in the final file. As @sritchie73 says this is fixed in PR #41.
